src/combobox/types.js

```diff
@@ -27,7 +29,7 @@ export type PropsT<OptionT = mixed> = {|
   // map whatever value the client gets into a visible string in the list item.
   mapOptionToString: OptionT => string,
   // Called when input value changes or option is selected.
-  onChange: string => mixed,
+  onChange: (string, ChangeSourceT) => mixed,
```

Comment:
Would you mind adding a comment for v10 to change the `onChange` type to be aligned with the rest of the components and use type
```
onChange: (params: {
  value: ValueT,
  type: ChangeActionT,
}) => mixed,
```